lib/gemstash/env.rb

```diff
@@ -177,5 +177,16 @@ def cache_client
         end
       end
     end
+
+    def storage_service
+      @storage_service ||= begin
+        case config[:storage_adapter]
+        when "local"
+          Storage::LocalService.new
+        else
+          raise "Invalid storage service: '#{config[:storage_adapter]}'"
+        end
```

Comment:
How about making this a bit more dynamic, maybe something like this:
```
config[:storage_adapter] = 'Storage::LocalService' # so a class name
```
```suggestion
        Object.const_get(config[:storage_adapter]).new
```

Of course, there needs to be some more checks and some form of `raise`, but it would make sense to make this less hard coded.

Reply:
It's okay for me, I can apply this change.

Reply:
I did the change :)